spec/features/layout_spec.rb

```diff
@@ -5,7 +5,10 @@
     visit '/'
 
     within 'head', visible: false do
-      expect(page).to have_xpath('/html/head/script[3]', text: 'UA-XXXXXXXXX-XX', visible: false)
+      gtm = page.find_all('script', visible: false).select do |node|
+        node[:src]&.match?(/googletagmanager.*id=UA-[X]{1,9}-[X]{1,2}/)
+      end
+      expect(gtm.count).to be >= 1
```

Comment:
:+1:

Reply:
@naseberry  for some reason this was UA-XXXXX-XX locally (5 middle x's) in some contexts but UA-XXXXXXXXX-XX(9 middle x's) in the ENV var `GA_TRACKING_ID`. could not find where this 'other' value (5 middle x's) was coming from. Any ideas?

Reply:
It may have been a mistake by me, the GA standard is 9 digits